spec.bs

```diff
@@ -3902,6 +3952,10 @@ The <dfn for=Navigator method>updateAdInterestGroups()</dfn> method steps are:
            [=interest group/trusted bidding signals slot size mode=] to |value|.
         1. Otherwise, set [=interest group/trusted bidding signals slot size mode=] to "none".
 
+        <dt>"`maxTrustedBiddingSignalsURLLength`"
+        1. If |value| is a {{long}} and |value| &ge; 0, set |ig|'s [=interest group/max trusted bidding signals url length=] to |value|.

```

Comment:
Wait, what else would it be besides a `{{long}}`?

Reply:
it can be anything, since it's parsed from a json for IG update. So we need to check if it's the expected data type. The same is done for all other update fields, for example, we check if [priority is a double](https://github.com/WICG/turtledove/blob/main/spec.bs#L3827).

Reply:
Added a suggestion https://github.com/WICG/turtledove/pull/1065#discussion_r1522008521